VimeoUpload/Upload/Networking/Shared/VimeoRequestSerializer+SharedUpload.swift

```diff
@@ -187,7 +187,7 @@ extension VimeoRequestSerializer
 
     func videoSettingsRequest(with videoUri: String, videoSettings: VideoSettings) throws -> NSMutableURLRequest
     {
-        guard videoUri.characters.count > 0 else 
+        guard videoUri.count > 0 else
```

Comment:
Should check `videoUri.isEmpty == false` here (guaranteed O(1) performance rather than O(n)).